### PR TITLE
Clarify that WKB means "standard" WKB enconding

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes https://github.com/opengeospatial/geoparquet/issues/14

### CODE REVIEW

File: format-specs/geoparquet.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Data that is better served in particular projections can choose to use an altern
 
 This is the binary format that the geometry is encoded in. The string 'WKB' to represent 
 [Well Known Binary](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary) is the only current option, but future versions
-of the spec may support alternative encodings.
+of the spec may support alternative encodings. This should be the ["standard"](https://libgeos.org/specifications/wkb/) WKB representation.
```
Comment (kylebarron) [inline format-specs/geoparquet.md:79]: Link to the `#standard-wkb` section?
```suggestion
of the spec may support alternative encodings. This should be the ["standard"](https://libgeos.org/specifications/wkb/#standard-wkb) WKB representation.
```

Maybe I missed this on the call, but that means we're sticking to 2D coordinates for 0.1?

**Review by pomadchin — COMMENTED** [review]
LGTM, (with the suggestion by @kylebarron). 
Also the commit message could be pretty confusing (i mean _Standard CRS for WKB_).

**Review by pomadchin — APPROVED** [review]

**Review by cholmes — APPROVED** [review]

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
>  Also the commit message could be pretty confusing (i mean "Standard CRS for WKB").

Squash and merge should have solved that!

**Comment by cholmes** [comment]
I noticed that squash and merge is the default and only enabled option now. Did someone change that? Or does github default to that now? Do we want squash & merge for all PR's? I'm happy for anything, just haven't done the squash and merge thing much before.

**Comment by jorisvandenbossche** [comment]
I changed that (but happy to change it back). I personally think that's the easiest option (and gives the cleanest git history)

**Comment by pomadchin** [comment]
I don't have strong feelings about it, but would prefer squash & merge as a default, it is easier to track the history of changes skipping extra commits.
Could be manually squashed if there are important commits to track in the history. ¯\_(ツ)_/¯

**Comment by cholmes** [comment]
> I changed that (but happy to change it back). I personally think that's the easiest option (and gives the cleanest git history)

Cool. More than happy to keep it, seems cool, I've just never worked with it.

**Pull request merged**